src/v2/api/controllers/GroupController.ts

```diff
@@ -24,8 +24,9 @@ import { GroupResponse, GroupsResponse } from '../responses/GroupResponse';
 import { GroupStudentsResponse } from '../responses/GroupStudentsResponse';
 import { CaptainResponse } from '../responses/CaptainResponse';
 import { ExtendDisciplineTeachersResponse } from '../responses/DisciplineTeachersResponse';
-import { ShortDisciplinesResponse } from '../responses/DisciplineResponse';
 import { ShortUsersResponse } from '../responses/UserResponse';
+import { QuerySemesterDTO } from '../dtos/QuerySemesterDTO';
+import {ShortDisciplineResponse, ShortDisciplinesResponse} from '../responses/DisciplineResponse';
```

Comment:
Remove unnecessary import - `ShortDisciplineResponse` (however, keep `ShortDisciplinesResponse`). Also add spaces after and before braces.